chords_usb.py

```
@@ -0,0 +1,156 @@
+            while retry_counter < self.retry_limit:
+                self.ser.write(b'WHORU\n')
+                try:
+                    response = self.ser.readline().strip().decode()
+                except UnicodeDecodeError:
+                    response = None
+
+                if response in self.supported_boards:
+                    self.board = response
+                    print(f"{response} detected at {port} with baudrate {baudrate}")
+                    self.num_channels = self.supported_boards[self.board]["Num_channels"]
+                    sampling_rate = self.supported_boards[self.board]["sampling_rate"]
```

Ruff (0.11.9)

Local variable `sampling_rate` is assigned to but never used

Remove assignment to unused variable `sampling_rate`

(F841)